README.md

```
@@ -17,13 +21,32 @@ To run the example project, clone the repo, and run `pod install` from the Examp
-it, simply add the following line to your Podfile:
+### CocoaPods
 
 ```ruby
 pod 'ZDMediator'
 ```
 
+### Swift Package Manager
+
+在 Xcode 中：`File` → `Add Package Dependencies...`，输入仓库地址：
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)